storybook/preview.js

```diff
@@ -84,8 +84,10 @@ export const globalTypes = {
 		toolbar: {
 			icon: 'outline',
 			items: [
-				{ value: 'none', title: 'None' },
-				{ value: 'wordpress-sidebar', title: 'WP Sidebar' },
+				{ value: 0, title: 'None' },
+				{ value: 248, title: 'WP Sidebar' },
+				{ value: 960, title: 'Large container' },
+				{ value: 600, title: 'Small container' },
```

Comment:
I'd suggest keeping the existing values and creating a map between string values and px values in the decorator itself. A good reason for that is that it will prevent such issues for users who have previously used Storybook and have an old cached value:

![Screenshot 2024-12-18 at 10 44 57](https://github.com/user-attachments/assets/abad0385-ec68-4c63-a7b0-b3fc45166113)

This is easy enough to repro once you set a max width, and then manually remove the global arg from the URL: `&globals=maxWidthWrapper:248`

Reply:
Precisely, thanks 👍